modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -229,6 +236,34 @@ export default class Ecdsa {
     return shares;
   }
 
+  /**
+   * Derives a child common keychain from common keychain
+   *
+   * @param commonKeychain - common keychain as a hex string
+   * @param path - bip32 path
```

Comment:
Kindly make the JSDOC statements follow the same convention as the other functions in the file
```suggestion
   * @param {string} commonKeychain - common keychain as a hex string
   * @param {string} path - bip32 path
```